thats_the_way.py

```diff
@@ -0,0 +1,17 @@
+from os import listdir
+from os.path import isfile, join
+
+
+def that_the_way(path):
+    file_names = [f for f in listdir(path) if isfile(join(path,f)) and 'deep' in f]
```

Comment:
f? is it a fly? is it a fruit? is it fresh? we cant know because you didn't give an indicative variable name :)

Reply:
כתבו פונקציה שמקבלת נתיב לתיקייה, ומחזירה את רשימת כל הקבצים **_שמתחילים_** ברצף האותיות "deep" באותה תיקייה.
בדקו שהפעלת הפונקציה על התיקייה images מחזירה שני קבצים.

אתה לא בודק שהקבצים מתחילים